tools/loader/ImageClassifier.cpp

```diff
@@ -83,11 +83,18 @@ llvm::cl::opt<ImageNormalizationMode> imageMode(
 llvm::cl::alias imageModeA("i", llvm::cl::desc("Alias for -image_mode"),
                            llvm::cl::aliasopt(imageMode),
                            llvm::cl::cat(imageLoaderCat));
+
+
+llvm::cl::opt<std::string> loadimageFormat(
+    "image_format",
+    llvm::cl::desc("Specify which image format to use"));
```

Comment:
All of our models up until now assume NCHW input -- can you add a default value here for NCHW?

Reply:
Indeed the current implementation is if you don't explicitly specify the imageformat, it does whatever is was earlier (NCHW). However this feedback has been given by most of you and hence I am working on making this change.